maxdiskusage.cc

```diff
@@ -79,7 +80,7 @@ maxdiskusage_notify(MYSQL_THD thd,
 
     if ((maxdiskusage_pct < 100) && (used_pct >= maxdiskusage_pct))
```

Comment:
Should this also compare against `maxdiskusage_block_pct`?
```suggestion
    if ((maxdiskusage_pct < 100) && ((used_pct >= maxdiskusage_pct) || used_pct > maxdiskusage_block_pct))
```